index.html

```diff
@@ -11,26 +11,29 @@
 
 <body>
   <section class="Container-1">
-    <main class="header-mobile">
-
+    
       <header class="menu1">
         <nav class="navigation">
           <a href="#" class="logo">My Logo</a>
-          <img src="media/Union.png" alt="" width="30px">
-          <ul>
-            <li>
-              <a href="">Portfolio</a>
+          <ul class="navigation-two">
+            <li class="nav">
+              <a href="#">Portfolio</a>
             </li>
-            <li>
-              <a href="">About</a>
+            <li class="nav">
+              <a href="#">About</a>
             </li>
-            <li>
-              <a href="">Contact</a>
+            <li class="nav">
+              <a href="#">Contact</a>
             </li>
           </ul>
+          <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
         </nav>
       </header>
-
+      <main class="header-mobile"></main>
```

Comment:
- [ ] I noticed that you're not using this `<main>` tag for any particular purpose, I suggest you delete it to keep the code clean and easy to read! 💯 🤓 🛠️